ui/components/app/data-deletion-error-modal/data-deletion-error-modal.test.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
```

Comment:
Can we also add a unit test that checks if there is a button and that `hideDataDeletionErrorModal` is called if the button is clicked?